src/templates/core/proposals/proposal_review_block.html

```diff
@@ -11,89 +11,98 @@
+                   '/email/proposal/{{ proposal.id }}/user/{{ review.user.id }}/',
+                   'proposal_{{ review.user.id }}'
+                   )"
            class="fa fa-envelope-o btn btn-xs btn-default">
         </a>
       </sup>
     </td>
-    <td>{{ reviewer.assigned }}</td>
+    <td>{{ review.assigned }}</td>
     <td>
-      {% if reviewer.accepted %}
-        Accepted: {{ reviewer.accepted }}
-      {% elif reviewer.declined %}
-        Declined: {{ reviewer.declined }}
+      {% if review.accepted %}
+        Accepted: {{ review.accepted }}
+      {% elif review.declined %}
+        Declined: {{ review.declined }}
       {% else %}
         Awaiting reply
       {% endif %}
     </td>
     <td>
-      {{ reviewer.due }}
-      <a href="{% url 'change_review_due_date' proposal.id reviewer.id %}">
+      {{ review.due }}
+      <a href="{% url 'change_review_due_date' proposal.id review.id %}">
         <i class="fa fa-edit">&nbsp;</i>
       </a>
     </td>
     <td>
-      {% if reviewer.completed %}
-        {{ reviewer.completed }}
+      {% if review.completed %}
+        {{ review.completed }}
       {% else %}
         --
       {% endif %}
     </td>
     <td>
-      {% if reviewer.completed %}
-        {{ reviewer.get_recommendation_display }}
+      {% if review.completed %}
+        {{ review.get_recommendation_display }}
       {% else %}
         --
       {% endif %}
     </td>
 
     <td>
-        {%  if reviewer.completed %}
-            <a href="{% url 'view_completed_proposal_review' proposal.id reviewer.id %}">
+        {%  if review.completed %}
+            <a href="{% url 'view_completed_proposal_review' proposal.id review.id %}">
                 View Review
             </a>
         {%  endif %}
     </td>
     <td>
-      {% if reviewer.reopened %}
-        {% if reviewer.completed %}
-          <a href="{% url 'reopen_proposal_review' proposal.id reviewer.id %}">
+      {% if review.reopened %}
+        {% if review.completed %}
+          <a href="{% url 'reopen_proposal_review' proposal.id review.id %}">
             Reopen
           </a>
         {% endif %}
       {% else %}
-        {% if reviewer.completed %}
-          <a href="{% url 'reopen_proposal_review' proposal.id reviewer.id %}">
+        {% if review.completed %}
+          <a href="{% url 'reopen_proposal_review' proposal.id review.id %}">
             Reopen
           </a>
         {% endif %}
       {% endif %}
     </td>
     <td width="10%">
-      <a href="{% url 'withdraw_proposal_review' proposal.id reviewer.pk %}"
-         class=" btn   {% if reviewer.withdrawn %}btn-info{% else %}btn-warning{% endif %} btn-sm pull-left"
+      <a href="{% url 'withdraw_proposal_review' proposal.id review.pk %}"
+         class=" btn
+                {% if review.withdrawn %}
+                    btn-info
+                {% else %}
+                    btn-warning
+                {% endif %}
+                    btn-sm pull-left"
```

Comment:
Not going to ask you to change it now, but having this indented one under `{% endif %}` is a little off putting